tb_plugin/test/test_profiler.py

```diff
@@ -1644,5 +1644,136 @@ def test_memory_view(self):
                 # op calls
                 self.assertEqual(mem_stat[name][6], values[6])
 
+    # Test group by "kernel detail + op name".
+    def test_group_by_kernel_columns(self):
+        json_content = """
+          [
+          {
+            "ph": "X", "cat": "Operator", 
+            "name": "op1", "pid": 13721, "tid": "123",
+            "ts": 200, "dur": 60,
+            "args": {"Input Dims": [[2, 8, 5], [], [], [], [], [], [], []], "External id": 3}
+          },
+          {
+            "ph": "X", "cat": "Operator", 
+            "name": "op2", "pid": 13721, "tid": "456",
+            "ts": 340, "dur": 70,
+            "args": {"Input Dims": [[], [32, 1000], [32], [], [], [], []], "External id": 4}
+          },          
+          {
+            "ph": "X", "cat": "Kernel", 
+            "name": "kernel1", "pid": 0, "tid": "stream 7",
+            "ts": 230, "dur": 15,
+            "args": {"correlation": 1000, "external id": 3, "device": 0,
+                    "grid": [16, 1, 1], "block": [16, 16, 16], "registers per thread": 18, "shared memory": 0}
```

Comment:
should we add test cases for missing these values?

Reply:
Manually removed a kernel's "block" field, and test pass on UI side:
![image](https://user-images.githubusercontent.com/10429114/125432896-3cfee6be-5cc8-4011-a28d-8124078c8c71.png)

Reply:
It would be better to make the edge case automatically

Reply:
Done. Added a unit test handling kernel without some field.